src/services/dashboards/dashboard-detail/store/dashboard-detail-info.ts

```diff
@@ -105,15 +103,9 @@ export const useDashboardDetailInfoStore = defineStore('dashboard-detail-info',
     }) as UnwrapRef<DashboardDetailInfoStoreState>;
     const validationState = reactive<ValidationState>({
         isNameValid: undefined,
-        isWidgetLayoutValid: computed(() => ({})), // is all widgets valid
-        widgetInheritVariablesValidMap: computed(() => {
-            const result = {};
-            // originState.dashboardWidgetInfoList.forEach(() => {
-            //     result[d.widget_key] = isWidgetValid(d, state.variables_schema);
-            // });
-            return result;
-        }),
-    });
+        isWidgetLayoutValid: computed(() => Object.values(validationState.widgetValidMap).every((d) => d === true)),
+        widgetValidMap: {},
+    }) as UnwrapRef<ValidationState>;
```

Comment:
issue: I think widgetValidMap must be reset on resetDashboardData() or setDashboardInfo().